meetings/2022-05-18.md

```diff
@@ -0,0 +1,120 @@
+* ...: There will be multiple data registries concerning multiple resource servers (storages).
+* ...: Alice will be able to delegate a grant she has received from Bob and allow Projectron to allow all or a subset of what Alice can access by Bob.
+* ...: The case of delegating access to data that one doesn't own is very interesting. The RO shouldn't care whom Alice delegates access to. However, then how can Bob's resource server take these authorizations into consideration?
+* ...: One of the problems we must solve: What if Alice delegates access to resources how can the authorization service in the security realm of Bob take this into account?
+* ...: If a Verifiable Credential were made available to the application, the VC can be presented through UMA claims pushing. Both ID token and VC of Delegated Data grant would have to be presented. This VC would have been signed by the authorization agent of Alice.
+* ...: When the authorizer is validating what access modes are available for Projectron, it would take into account the Data Grant of Bob to Alice as well as the Delegated Data grant, which could be a subset of what was granted to Alice.
+* Laurens: Very explicit mechanism for delegation through the use of VCs is a plus.
+* Pavlik: With longer delegation chains some problems could occur, what do we expose and how do we handle chained signatures. However, the current proposal is what we should support as a minimum to have an end-user share a resource to which they were granted access but which they do not own.
+* ...: Discovery is also very important, as it is intertwined with authorization. It is a prerequisite for discovery.
+* ...: Shows [discussion #203](https://github.com/solid/authorization-panel/discussions/203), with diagrams of an Access Matrix. Where different views on authorization rules are visualized, i.e. by column, by row.
+* ...: Data grants are similar to the collapse by row, i.e. per-agent.
+* ...: By saying what can be accessed the agent (through the data grant), it gets a discovery path that could be a replacement of the WAC-Allow header.
+* ...: Discovery starting from the resources (collapse by column) makes less sense as a discovery mechanism.
+* Mathieu: Agrees that discoverability is important.
+* Pavlik: Do we have consensus that authorization is prerequisite to discovery? (Public resources are a separate case.)
+* Mathieu: Yes.
+* Laurens: Agreed.
+* Pavlik: This is something that I have not seen addressed up until now. In the Solid Application Interoperability specification the guiding principle has been that of least-privilege.
+* ...: Without any consent or trust chain the existence of data shouldn't be discoverable.
+* ...: To conclude we should look at (1) reconciling the work of ACP and Interop and (2) define a generalized view of the Access Matrix, where data grants give a collapse by agent and ACP gets a collapsed view by resource. I believe this will make more sense for discovery.
```

Comment:
```suggestion
* ...: Without any consent or trust chain, the existence of data shouldn't be discoverable.
* ...: To conclude, we should look at (1) reconciling the work of ACP and Interop, and (2) define a generalized view of the Access Matrix, where data grants give a collapse by agent, and ACP gets a collapsed view by resource. I believe this will make more sense for discovery.
```